### ISSUE TITLE
[Feature Request] Metadata Explorer Runtime Casing

### ISSUE THREAD

**Issue by gilescope** [w3f/polkadot-wiki#4517]
![image](https://user-images.githubusercontent.com/803976/221614044-8845c130-7c9f-4197-aa4c-f345c5a59793.png)

At the moment if someone tries to call a runtime method with the above using `State` `call` they will find that they need to have the method in snake case rather than in camel case. It would be helpful if we displayed it in the metadata explorer in snake case to give them this visual cue. E.g. `AccountNonce` should be `account_nonce` (This is only for the runtime calls)

**Comment by gilescope** [comment]
Maybe we should add an attribute like `State.call method: AccountNonceApi_account_nonce` and `State.call method: BlockBuilderApi_check_inherents`? I don't know if there's a better way to publicise this format but it's not something that someone would guess at so it's important that we make it clear to people.

**Comment by alfarok** [comment]
Hey @gilescope, thanks for checking out the new tool and providing feedback!  The current camel casing is reflective of the runtime calls in polkadotjs.  I see what you are saying though regarding the difference in casing in the runtime spec.

One solution could be to explicitly state that the calls are for the PolkadotJS api, such as:

`PolkadotJS API Endpoint: api.call.accountNonceApi.accountNonce`

Then also add another row showing the original runtime formatting, such as:

`Runtime Method: AccountNonceApi_account_nonce`

Let me know if I am understanding the issue correctly and if you have any additional thoughts on the solution above.  Thanks again!

**Comment by gilescope** [comment]
Yeah that would be helpful. I like the idea of showing the js call too.
